src/content/details/faqs.mdx

```diff
@@ -11,22 +11,21 @@ index: 2000
 
 **What is Neural Magic?**
 
-Founded by a team of award-winning MIT computer scientists and funded by Amdocs, Andreessen Horowitz, Comcast Ventures, NEA, Pillar VC, and
-Ridgeline Partners, Neural Magic is the creator and maintainer of the Deep Sparse Platform. It has several components, including the
-[DeepSparse Engine,](/products/deepsparse) a CPU runtime that runs sparse models at GPU speeds. To enable companies the ability to use
+Neural Magic was founded by a team of award-winning MIT computer scientists and is funded by Amdocs, Andreessen Horowitz, Comcast Ventures, NEA, Pillar VC, and
+Ridgeline Partners. It has several components, including [DeepSparse,](/products/deepsparse) a CPU runtime that runs sparse models at GPU speeds. To enable companies the ability to use
 ubiquitous and unconstrained CPU resources, Neural Magic includes [SparseML](/products/sparseml) and the [SparseZoo,](/products/sparsezoo)
 open-sourced model optimization technologies that allow users to achieve performance breakthroughs, at scale, with all the flexibility of software.
 
-**What is the DeepSparse Engine?**
+**What is DeepSparse?**
 
-The DeepSparse Engine, created by Neural Magic, is a general purpose engine for machine learning, enabling machine learning to be practically
+DeepSparse, created by Neural Magic, is a general purpose engine for machine learning, enabling machine learning to be practically
```

Comment:
@rsnm2 Missed this out. We need our new boiler plate PMM language where this is describing a suite of components including DeepSparse Benchmark, DeepSparse Engine, DeepSparse Pipeline, and DeepSparse Server, that allows the users to xxxx

Reply:
Fixed this